documentation/index.js

```diff
@@ -0,0 +1,42 @@
+const swaggerJsDoc = require('swagger-jsdoc');
+const swaggerui = require('swagger-ui-express');
```

Comment:
Great work Issa! I think its great practise to use ES6 syntax.
and make sure to stop tracking your `.env` file

Reply:
yeah let me work on that

Reply:
In documentation I didn't use es6 cause I didn't put it in src and I'm building source, so I used require to make it readable by node